tests/test_cleaner.py

```diff
@@ -0,0 +1,213 @@
+#
+# LSST Data Management System
+#
+# Copyright 2008-2019  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import datetime
+import os
+import time
+import lsst.ctrl.oods.directoryScanner as ds
+from lsst.ctrl.oods.cacheCleaner import CacheCleaner
+import lsst.utils.tests
+import tempfile
+
+
+def setup_module(module):
```

Comment:
Should be at the end, and should have an `if __name__ == "__main__"` block.  See https://developer.lsst.io/python/testing.html#memory-and-file-descriptor-leak-testing